src/patient-search/patient-search-result/patient-search-result.component.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styles from "./patient-search-result.component.css";
+import dayjs from "dayjs";
+
+export default function PatientSearchResults(props: PatientSearchResultsProps) {
+  return props.patients.map((patient, index) =>
+    renderPatient(patient, index + 1)
+  );
+
+  function renderPatient(patient, resultNumber) {
+    const preferredIdentifier =
+      patient.identifiers.find(i => i.preferred) || patient.identifiers[0];
+
+    return (
+      <div
```

Comment:
I'm thinking this should be a `<Link>` right? Taking you to the patient chart? Probably with `className="unstyled"`

Reply:
you're right